openedx_tagging/core/tagging/rest_api/paginators.py

```diff
@@ -0,0 +1,66 @@
+from edx_rest_framework_extensions.paginators import DefaultPagination  # type: ignore
```

Comment:
Nit: It's better to use `# type: ignore[import]` so that when `edx_rest_framework_extensions` implements type hints, mypy will give us a warning that this `ignore` is no longer needed, and we can remove it to get the type hints working. If you just use `type: ignore` it will be ignored forever, even if `edx_rest_framework_extensions` implements type hints at some point.

Reply:
Noted, thanks!

Reply:
Updated here: https://github.com/openedx/openedx-learning/pull/78/commits/9a15562c8509eee795dff0a939a9310b8a33af3f. I updated more of `type: ignore` in the code